ui/app/appointments/app.js

```diff
@@ -79,7 +79,9 @@ angular
                 url: '/calendar',
                 tabName: 'calendar',
                 params: {
-                    viewDate: null
+                    viewDate: null,
+                    getAppointments: true,
+                    appointmentData: null
```

Comment:
If this is data for multiple appointments then make it plural: `appointmentsData `.